Fix-#1392 Updated Settings Toolbar Title on Back Button Pressed

rutvik-panchal opened the pull request:
Fixes #1392 
Updated Toolbar Title on the back button pressed from the change password window.

**Screenshots:**

![ezgif-3-42bb9c5a8e1d](https://user-images.githubusercontent.com/39809059/76164066-dc184c00-6171-11ea-8bc0-37343ac7b7c7.gif)
Updated Toolbar Title on the back button pressed from the change password window.
ing them.

rutvik-panchal committed:
Fix-#1392

Updated Toolbar Title on back button pressed from change password window.

jawidMuhammadi commented on app/src/main/java/org/mifos/mobile/ui/fragments/SettingsFragment.java:
```diff
@@ -36,6 +36,7 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
     @Override
     public void onResume() {
         super.onResume();
+        getActivity().setTitle("Settings");
```
Please remove hardcoded "Settings" strings from here. Using hardcoded strings is not a good practice. First, add the string in the `strigs.xml` file then use it dynamically.

rutvik-panchal replied:
Yes, I Will take care of that in the future & will update it soon.

rutvik-panchal committed:
Fix #1392

updated the hardcoded string

rutvik-panchal committed:
Fix #1401 : Fix for QR code icon color

The qr code icon inside SavingsAccountDetails and LoanAccountDetails fragment had different color than rest of the icons displayed on the screen.
Added ic_qrcode_scan_black for Add Beneficiary Options Icon.

rutvik-panchal committed:
Revert "Fix #1401 : Fix for QR code icon color"

This reverts commit bcfc748e6241aa232d9690000b0168aca025d205.
